code/modules/client/client defines.dm

```diff
@@ -20,6 +20,13 @@
 
 	var/adminhelped = 0
 
+	var/gc_destroyed //Time when this object was destroyed.
```

Comment:
I'm guessing this isn't actually needed, but I included it just in case; we should probably make clients a child of `/datum` some time.